lib/types.d.ts

```diff
@@ -11,10 +11,13 @@ export type DocNode =
   | DocNodeInterface
   | DocNodeImport;
 
+export type DeclarationKind = "private" | "export" | "import" | "declaration";
```

Comment:
Having a comment here explaining what each of these mean would be very useful. Does `"declaration"` mean the `declare` keyword in TS?

Reply:
Hmmm... yeah, maybe `declare` is better, yes that is what it means.  I will change it to `declare` and add some comments.